src/panels/Scene/SceneTree/SceneTreeFilters.tsx

```diff
@@ -48,50 +49,53 @@ export function SceneTreeFilters({ setFilter, filter }: Props) {
           </ActionIcon>
         </Menu.Target>
         <Menu.Dropdown maw={'300px'}>
```

Comment:
```suggestion
        <Menu.Dropdown maw={'330px'}>
```
to fix that info box jumps down to next level